uwblueprint/supportive-housing
# Spinner

## Notion task link
n/a


## Implementation description
* Before, the header of the table would appear on the screen before the table data was available. Table data would appear at a delay.
* adding a spinner if we're in the process of fetching table data
* adding a "no results found" message if no results are founf



## Steps to test
1. go to home page
2. check if the spinner is there and looks good while the table is loading
3. make sure the table loads
4. filter so that there are 0 results and check if it gives a message that says "No results found."
5. repeat for resident and employee directory pages (step 4 not necessary)

## Checklist
